### ISSUE TITLE
Support new notification type: `severed_relationships`

### ISSUE THREAD

**Issue by shuuji3** [elk-zone/elk#3077]
The new notification type `severed_relationships` was introduced by the recent Mastodon release v4.3 (on Oct 2024). This causes the debug message on the notification tab.

ref. Notification - Mastodon documentation - https://docs.joinmastodon.org/entities/Notification/#type

Original report: https://github.com/elk-zone/elk/discussions/3076

**Comment by ayoayco** [comment]
Ah we might also need to prevent this in the future by gracefully ignoring if servers send notification types elk doesn’t know 🤔

**Comment by shuuji3** [comment]
Agreed, we should log messages like this to the console, instead of the user-facing place.

I got another notification type `aanual_report` today 😅 

![screenshot of the notification tab showing `annual_report` type is not supported](https://github.com/user-attachments/assets/1f77c51c-b6ce-44d7-9c3a-7dfc38e0342b)

**Comment by ayoayco** [comment]
Hey @shuuji3 shall we close this one? :)

**Comment by shuuji3** [comment]
Sure! Thanks for checking.
